sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/RedundantReturnCheck.java

```diff
@@ -364,11 +364,9 @@ else if (isFinalReturn(toVisitAst)) {
                 toVisitAst = returnAst;
             }
 
-            toVisitAst = SevntuUtil.getNextSubTreeNode(toVisitAst, currentNodeAst);
+            toVisitAst = SevntuUtil.getNextSubTreeNode(toVisitAst, lastStatementInCatchBlockAst);
         }
 
-        currentNodeAst = SevntuUtil.getNextSubTreeNode(currentNodeAst,
```

Comment:
This line removed as `getNextSubTreeNode` is read-only method and this statement does nothing.
PMD complains for unused variable here.